parse_nucleotide_fasta.c

```diff
@@ -0,0 +1,263 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <getopt.h>
+#include <string.h>
+
+typedef struct Flags {
+    int flag_raised;  // 1 if raised, 0 if not.
```

Comment:
Oh god, is there seriously not a bool in c? stdbool.h is not available?

https://www.geeksforgeeks.org/bool-in-c/